src/Employer/Employer.Web/Mappings/VacancyDisplayViewModelMapper.cs

```diff
@@ -43,6 +43,7 @@ public async Task MapFromVacancyAsync(DisplayVacancyViewModel vm, Vacancy vacanc
             vm.ApplicationUrl = vacancy.ApplicationUrl;
             vm.CanDelete = vacancy.CanDelete;
             vm.CanSubmit = vacancy.CanSubmit;
+            vm.CanReadyForReview = vacancy.CanReadyForReview;
```

Comment:
could this property be named `IsReadyForReview` or `CanReview`

Reply:
I have changed to CanReview